tests/kafkatest/benchmarks/streams/streams_simple_benchmark_test.py

```diff
@@ -93,11 +95,18 @@ def test_simple_benchmark(self, test, scale):
             node[num] = self.driver[num].node
             node[num].account.ssh("grep Performance %s" % self.driver[num].STDOUT_FILE, allow_fail=False)
             data[num] = self.driver[num].collect_data(node[num], "" )
-                
+            self.driver[num].read_jmx_output_all_nodes()
+
 
         final = {}
         for num in range(0, scale):
             for key in data[num]:
                 final[key + str(num)] = data[num][key]
-        
+
+            for key in sorted(self.driver[num].jmx_stats[0]):
+                self.logger.info("%s: %s" % (key, self.driver[num].jmx_stats[0][key]))
+
+            final["jmx-avg" + str(num)] = self.driver[num].average_jmx_value
+            final["jmx-max" + str(num)] = self.driver[num].maximum_jmx_value
```

Comment:
Just for my own education: average and max of what?

Reply:
This is to calculate the avg and max of all the collected metrics. In this PR I'm collecting poll / process / commit rate and latency, and compute the avg / max of all of them

In the follow-up PR (https://github.com/apache/kafka/pull/4744) I'm collecting consumer metrics for comparison.

Sorry I have do do this series in not-so-independent PRs, as I do not want to add a too large one, but separating them into steps is also a bit tricky.

I have also updated the description of this PR. Hope that helps understanding this change better.